### PR TITLE
win_virtio_serial_data_transfer_reboot: cleanup

### PR DESCRIPTION
**Pull request by PandaWei**
fix assigned but never used local variable 'ports' and 'd_file'.

Signed-off-by: Wei Jiangang <weijg.fnst@cn.fujitsu.com>

### CODE REVIEW

File: qemu/tests/win_virtio_serial_data_transfer_reboot.py [hunk 1]
```diff
@@ -32,15 +32,13 @@ def get_virtio_port_host_file(vm, port_name):
...
         """
-        ports = []
         for port in vm.virtio_ports:
             if isinstance(port, qemu_virtio_port.VirtioSerial):
                 if port.name == port_name:
                     return port.hostfile
 
     def receive_data(session, serial_receive_cmd, data_file):
         output = session.cmd_output(serial_receive_cmd, timeout=30)
-        d_file = os.path.join(data_dir.get_deps_dir("win_serial"), data_file)
```
Comment (PandaWei) [inline qemu/tests/win_virtio_serial_data_transfer_reboot.py]: And the data_file is absolute path. which is initialized in function run().

**Comment by PyLearner** [comment]
Acked-by : Meng Yang meyang@redhat.com

**Comment by xutian** [comment]
Merged, thanks @PandaWei

**Comment by xutian** [comment]
Merged, thanks @PandaWei

**Pull request merged**